docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -0,0 +1,174 @@
+* Update Auditor route tables with Ledger and Client VPC peering connections based on [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
+* Update Client route tables with Ledger and Auditor VPC peering connections based on [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
+* Add new Inbound and Outbound rule to restrict unwanted access to Ledger and Auditor.
+    * Add new Inbound rules to the Ledger Network ACLs to allow the Ledger Envoy LoadBalancer (50051, 50052) access from the Auditor and Client.
+        * You must set high priority to this rule.
+    * Add new Inbound rules to the Ledger Network ACLs to allow ssh access from the internet.
+        * You must set second priority to this rule.
+    * Add new Inbound rules to Ledger Network ACLs to allow all access to Ledger VPC.
+        * You must set third priority to this rule.
+    * Add new Outbound rules to the Ledger Network ACLs to allow all access to the internet.
+    * Add Ledger private subnets to subnet associations of the Ledger Network ACLs.
+    * Add new Inbound rules to the Auditor Network ACLs to allow the Auditor Envoy LoadBalancer (40051, 40052) access from the Ledger and Client.
+        * You must set high priority to this rule.
+    * Add new Inbound rules to the Auditor Network ACLs to allow ssh access from the internet.
+        * You must set second priority to this rule.
+    * Add new Inbound rules to Auditor Network ACLs to allow all access to Auditor VPC.
+        * You must set third priority to this rule.
+    * Add new Outbound rules to the Auditor Network ACLs to allow all access to the internet.
+    * Add Auditor private subnets to subnet associations of the Auditor Network ACLs.
+
+Note: - We expect you have created the Client VPC for your application deployment.
+
+## Step 3. Install Scalar DL Auditor
+
+### Prerequisites
+
+You must install Helm on your bastion to deploy [helm-charts](https://github.com/scalar-labs/helm-charts):
+
+* [Helm](https://helm.sh/docs/intro/install/): Helm command-line tool to manage releases in the EKS cluster, Helm version 3.2.1 or latest is required. In this guide, it is used to deploy Scalar DL Auditor and Schema loading helm charts to the EKS cluster.
+* You must create a Github Personal Access Token (PAT) on the basis of Github official document with `read:packages` scope, it is used to access the `scalar-auditor` and `scalardl-schema-loader` container images from GitHub Packages.
+
+### Requirements
+
+* You must have the authority to pull `scalar-auditor` and `scalardl-schema-loader` container images.
+
+### Recommendations
+
+* You should confirm that the replica count of the Auditor and Envoy pods in the `scalardl-audit-custom-values.yaml` file is equal to the number of nodes in the `scalardlpool`.
```

Comment:
These two are saying the same thing?
Also, `you should confirm`  sounds very weird. If we should set, then say so. Only confirming is recommended?

Moreover, we should talk about the reason for the recommendation.

Reply:
> These two are saying the same thing?

No, the First one is for avoiding resource shortage.
Second one for better performance (If we use 3 Ledger recommended to use 3 auditors)